src/adapter/src/coord/sequencer.rs

```diff
@@ -1142,6 +1142,18 @@ impl<S: Append + 'static> Coordinator<S> {
                     DEFAULT_LOGICAL_COMPACTION_WINDOW_MS,
                 )
                 .await;
+
+                // Advance the new table to a timestamp higher than the current read timestamp so
+                // that the table is immediately readable.
+                let upper = self.get_local_read_ts().step_forward();
+                let appends = vec![(table_id, Vec::new(), upper)];
+                self.controller
```

Comment:
This will cause `CREATE TABLE` to get a bit slower now because it'll block on a persist consensus operation, right? I guess that's ok?

Why isn't the timestamp `since_ts.step_forward()`?

Reply:
> This will cause CREATE TABLE to get a bit slower now because it'll block on a persist consensus operation, right? I guess that's ok?

It will cause the `CREATE TABLE` operation to be slower, but the total time from typing `CREATE TABLE` to being able to read the new table is now smaller. Though the more likely use case is to insert after creating a table which will now be slower.

> Why isn't the timestamp since_ts.step_forward()?
  
Good point, that would actually be a better choice. Just pushed that update.